virttest/utils_time.py

```diff
@@ -0,0 +1,229 @@
+import logging
+import re
+
+from avocado.utils import process
+
+from virttest import error_context
+
+
+def get_host_timezone(params):
```

Comment:
Having discussed with @luckyh , it's better not to take `params` as an argument.
You can just put the cmd in your func body, please try cover as many distributions as possible.